letta/schemas/tool_rule.py

```diff
@@ -21,6 +21,17 @@ class ChildToolRule(BaseToolRule):
     children: List[str] = Field(..., description="The children tools that can be invoked.")
 
 
+class ConditionalToolRule(BaseToolRule):
+    """
+    A ToolRule that conditionally maps to different child tools based on the output.
+    """
+    type: ToolRuleType = ToolRuleType.conditional
+    default_child: str = Field(..., description="The default child tool to be called")
+    child_output_mapping: Dict[Union[bool, str, int], str] = Field(..., description="The output case to check for mapping")
+    children: List[str] = Field(..., description="The child tool to call when output matches the case")
```

Comment:
I'm confused what this field is - isn't the child tool specified in `child_output_mapping`?

Reply:
yeah this is repetitive. removed `children`